tests/integration/extenders/ModelTest.php

```diff
@@ -261,6 +322,31 @@ public function custom_default_attribute_is_inherited_to_child_classes()
         $this->assertEquals(42, $post->answer);
     }
 
+    /**
+     * @test
+     */
+    public function custom_default_attribute_inheritance_prioritizes_child_class()
+    {
+        $this->extend(
+            (new Extend\Model(AbstractModel::class))
+                ->default('answer', 'dont do this'),
```

Comment:
Same here. This scenario is useful with the `Post` / `CommentPost` combination, but IMO we don't need the `AbstractModel` example.

Reply:
Clarified the use case